tasks/release.py

```diff
@@ -1340,3 +1341,64 @@ def check_previous_agent6_rc(ctx):
         payload = {'message': err_msg}
         send_slack_msg(ctx, payload, os.environ.get("SLACK_DATADOG_AGENT_CI_WEBHOOK"))
         raise Exit(message=err_msg, code=1)
+
+
+@task
+def pin_integrations_core(ctx, slack_webhook=None):
+    """
+    Create a PR to bump the integrations core fields in the release.json file
+    """
+    if os.environ.get("GITHUB_ACTIONS"):
+        set_git_config('user.name', 'github-actions[bot]')
+        set_git_config('user.email', 'github-actions[bot]@users.noreply.github.com')
+
+    commit_hash = (
+        ctx.run(
+            r'git ls-remote https://github.com/DataDog/integrations-core HEAD',
+            hide=True,
+        )
+        .stdout.strip()
+        .split()[0]
+    )
+
+    current = current_version(ctx, 7)
+    current.rc = False
+    current.devel = False
```

Comment:
If I understand things correctly, you need to set this to False so that `create_release_pr` correctly finds the labels for the latest dev branch, but that is counter intuitive since we're only doing this for the development branch.
I'd add a comment to avoid potential confusion